go/iot/main.go

```diff
@@ -16,37 +16,46 @@
 package main
 
 import (
+	"context"
 	"time"
+	"log"
 
-	"github.com/mgutz/logxi/v1"
+	"contrib.go.opencensus.io/exporter/stackdriver"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
 	"gobot.io/x/gobot"
 	"gobot.io/x/gobot/drivers/gpio"
 	"gobot.io/x/gobot/platforms/raspi"
 )
 
 const (
-	high int = 1
-	low  int = 0
+	high = 1
+)
+
+var (
+	gpioVoltage = stats.Int64("my.org/measure/gpio_voltage_level", "level of voltage", stats.UnitDimensionless)
 )
 
 func main() {
+	ctx := context.Background()
+	// TODO: It takes around one minute to detect the full edge of voltage change. Needs to tune the report period
+	initOpenCensus("opencensus-java-stats-demo-app", 1)
```

Comment:
You should detect the project id from env vars, instead of hard coding here.

Reply:
okay